101-app-service-create-managed-certificate-apex/metadata.json

```diff
@@ -0,0 +1,14 @@
+
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+   "itemDisplayName":"Create a App Services Managed Certificate",
+   "description":"Create Managed Certificate (Free) APEX (root domain) for WebApp",
+   "summary":"Create Managed Certificate (Free) APEX (root domain) for WebApp",
+   "githubUsername":"coelho5br",
+   "dateUpdated":"2021-05-06",
+   "validationType": "Manual",
```

Comment:
I'm not seeing anything that requires manual validation - remove this when you add the prereq template

Reply:
@bmoore-msft 
I'll leave as manual for now, due to dependency on the A record to match where the Web App will be created. If the deployment has access to the Azure DNS zone holding the domain we will run the test, I can add it to the prereq. I only need to know the Resource group of the DNS zone